onnx/defs/function.h

```diff
@@ -57,6 +56,12 @@ class FunctionBuilderRegistry {
   static Common::Status function_builder_##counter##_status = \
       FunctionBuilderRegistry::OnnxInstance().Register(function_builder);
 
+// Method docomposing all functions in graph
+Common::Status DecomposeGraph(
+    GraphProto& g,
+    const std::string& domain,
+    std::vector<std::string> function_list = {});
```

Comment:
std::vector<std::string> -> const std::vector<std::string>&   ?

meanwhile, domain and function name may be put together? otherwise, what if you want to inline functions from different domains?

Reply:
I think this should be a separate PR, as we change the API to retrieve function with both version and domain. Current one only takes domain